helm-charts/sawmills-collector/tests/telemetry_keda_overlay_test.yaml

```
@@ -102,6 +102,24 @@ tests:
           path: data["keda.yaml"]
           pattern: '(?m)^\s*metrics/keda:\s*$'
 
+  - it: keeps central queue metrics in the default keda telemetry overlay
+    template: telemetry-keda-configmap.yaml
+    set:
+      kedaScaler.enabled: true
+    asserts:
+      - matchRegex:
+          path: data["keda.yaml"]
+          pattern: 'otelcol_loadbalancer_central_queue_compressed_bytes'
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)